lib/ZcapClient.js

```diff
@@ -92,13 +102,16 @@ export class ZcapClient {
    * @param {string|Array} [options.allowedActions] - Optional list of allowed
    *   actions or string specifying allowed delegated action. Default: [] -
    *   delegate all actions.
+   * @param {Function<LinkedDataSignature>} [options.suite] - Optional signature
+   *   suite to use when digitally signing the delegation. Default: The
+   *   signature suite that was used when initializing the ZcapClient.
    *
    * @returns {Promise<object>} - A promise that resolves to a delegated
    *   capability.
    */
   async delegate({
     url, capability, targetDelegate, invocationTarget, expires,
-    allowedActions = []
+    allowedActions = [], suite
```

Comment:
Perhaps this should be an *instance* of a suite? Is it safe to assume that all suite instances can be constructed with only a `signer` and `verificationMethod`? Alternatively, we could accept either a `suiteClass` or `suite` here.